cut-release-pr.sh

```diff
@@ -0,0 +1,128 @@
+#!/bin/bash
```

Comment:
Having this as a bash script is pretty clunky. It would be nice if this was a github action workflow. I think that would take a bit more work. Can iterate on it if this is a direction we want to go.